### ISSUE TITLE
Example error?

### ISSUE THREAD

**Issue by csullens-work** [terraform-aws-modules/terraform-aws-redshift#6]
Hi,

I'm very new to Terraform, and want to add my AWS Redshift to the mix. I'm trying to "plan" the very basic example and I'm hitting the error:

`* module.redshift.output.this_redshift_cluster_vpc_security_group_ids: Resource 'aws_redshift_cluster.this' does not have attribute 'vpc_security_group_ids' for variable 'aws_redshift_cluster.this.vpc_security_group_ids'`

Any ideas?

thanks,
Chris

**Comment by antonbabenko** [comment]
Probably example has to be updated, pull-request is welcome.

**Comment by jaypatrickhoward** [comment]
Only happens when no value is supplied for vpc_security_group_ids.  I edited main.tf and hard-coded a dummy value (["abc"]) and no longer get the error.

**Comment by jaypatrickhoward** [comment]
Think it may have to do with the count=0 behavior introduced in terraform 0.11:

https://www.terraform.io/upgrade-guides/0-11.html

I'm using 0.11.

**Comment by antonbabenko** [comment]
@csullens-work Thanks for opening this issue. Apparently, an example was not in a good shape.

<p>
<details>
<summary>Indeed `vpc_security_group_ids` is required.</summary>

```
* module.redshift.aws_redshift_cluster.this: 1 error(s) occurred:

* aws_redshift_cluster.this: [WARN] Error modifying Redshift Cluster (my-cluster): InvalidParameterValue: The vpc-security-group-ids parameter must have at least one argument.
	status code: 400, request id: 617eaafa-b030-11e8-ae7a-8f21064be24c
```

</details>
</p>

I have updated the code in 4ba92b6 , tested it and released [v1.4.0](https://github.com/terraform-aws-modules/terraform-aws-redshift/releases/v1.4.0).
